modules/zookeeper/src/test/java/org/apache/ignite/spi/discovery/zk/ZookeeperDiscoverySpiTestSuite4.java

```diff
@@ -42,7 +43,8 @@
     GridCacheReplicatedAtomicMultiNodeFullApiSelfTest.class,
     IgniteCacheReplicatedQuerySelfTest.class,
     DistributedMetaStorageTest.class,
-    DistributedMetaStoragePersistentTest.class
+    DistributedMetaStoragePersistentTest.class,
+    ZookeeperNodeValidatedEventTest.class
```

Comment:
The `IgniteNodeValidatedEventTest` class can be added here. `ZookeeperNodeValidatedEventTest` is redundant.